test/Microsoft.IdentityModel.JsonWebTokens.Tests/JwtTokenUtilitiesTests.cs

```diff
@@ -13,6 +14,26 @@ namespace Microsoft.IdentityModel.JsonWebTokens.Tests
 {
     public class JwtTokenUtilitiesTests
     {
+        [Fact]
+        public void ClaimTypeMappingIsIndependent()
+        {
+            // Each handler should have its own instance of the ClaimTypeMap
+            var jwtClaimsMapping = JwtSecurityTokenHandler.DefaultInboundClaimTypeMap;
+            var jsonClaimsMapping = JsonWebTokenHandler.DefaultInboundClaimTypeMap;
+
+            Assert.NotEmpty(jwtClaimsMapping);
+            Assert.NotEmpty(jsonClaimsMapping);
+
+            Assert.Equal(jwtClaimsMapping, jsonClaimsMapping);
```

Comment:
What does Assert.Equal do here?
Our model to check if two dictionaries are the same it to use IdentityComparer.[AreStringDictionariesEqual](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/2b1b78bf38a5739719e95b7045d81902a2cf09d5/test/Microsoft.IdentityModel.TestUtils/IdentityComparer.cs#L736).